samples/highcharts/accessibility/point-description-enabled-threshold/demo.js

```diff
@@ -0,0 +1,48 @@
+Highcharts.chart('container', {
+    chart: {
+        type: 'line'
+    },
+    title: {
+        text: 'Daily Steps Count'
+    },
+    xAxis: {
+        categories: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'],
+        accessibility: {
+            description: 'Day of the week'
+        }
+    },
+    yAxis: {
+        title: {
+            text: 'Steps'
+        }
+    },
+    accessibility: {
+        series: {
+            pointDescriptionEnabledThreshold: 1
```

Comment:
This might be an error. I still get picture info for each point during navigation (in Safari).

Reply:
It is now only happening for the first point of the first series after the end of the first series walk and caused by the tooltip highlight.

Reply:
I am able to reproduce this in Safari + VoiceOver. In Chrome + VoiceOver it works like expected. Will file bug report for it.